ui/temboardui/static/src/settings.users.js

```diff
@@ -14,7 +14,18 @@ createApp({
   },
   created() {
     this.$nextTick(() => {
-      new DataTablesLib("#tableUsers", { stateSave: true });
+      new DataTablesLib("#tableUsers", {
+        stateSave: true,
+        columns: [
+          { width: "auto" }, // Username
+          { width: "auto" }, // Email
+          { width: "12rem" }, // Phone
+          { width: "6rem" }, // Active
+          { width: "6rem" }, // Admin
+          { width: "auto" }, // Groups
+          { width: "6rem" }, // Actions
+        ],
+      });
     });
   },
-}).mount("#app");
+}).mount("#vueapp");
```

Comment:
These changes don't seem related to the switch to Flask. Can you put them in a separate commit?

Also is the change from `#app` to `#vueapp` required? Or is it just a matter of taste?

Reply:
C'est juste de l'harmonisation avec les autres pages.

Reply:
Peux-en faire un commit séparé ?